### PR TITLE
small changes in imports and ajust to pep8

### PR DESCRIPTION
**Pull request by luroche**
approve me now!!

### CODE REVIEW

**Review by mariuscrsn — CHANGES_REQUESTED** [review]

File: code/drv/drv_scpi/drv_scpi.py [hunk 1]
```diff
@@ -146,7 +142,7 @@ def receive_msg(self) -> List[str]:
         Raises:
             - None
         '''
-        msg = self.__serial.read_until(self.__separator)
+        msg = self.__serial.readall()
```
[the remark this answers is not in the document]
Comment (luroche) [reply code/drv/drv_scpi/drv_scpi.py:145]: For example, in the multimeter, if the complet message is: 'FETCH?'\n'1.19939508e1', using read_until(b'\n') only read FETCH?.

[the remark this answers is not in the document]
Comment (mariuscrsn) [reply code/drv/drv_scpi/drv_scpi.py:145]: It must be handler in the upper module (`drv_bk`).

File: code/drv/drv_scpi/example.py [hunk 2]
```diff
@@ -1,33 +1,27 @@
...
 
-#######################          MODULE IMPORTS          #######################
-from drv.drv_scpi import DrvScpiHandlerC
+#######################      SYSTEM ABSTRACTION IMPORTS  #######################
+sys.path.append(os.getcwd()+'\\code')  #get absolute path
+# from sys_abs.sys_log import sys_log_logger_get_module_logger
+# if __name__ == '__main__':
+#     from sys_abs.sys_log import SysLogLoggerC
+#     cycler_logger = SysLogLoggerC('./sys_abs/sys_log/logginConfig.conf')
+# log = sys_log_logger_get_module_logger(__name__)
```
Comment (RobertoAldea) [inline code/drv/drv_scpi/example.py:19]: When uploading try to uncomment neccesary code in future

File: code/drv/drv_scpi/example.py [hunk 3]
```diff
@@ -36,19 +30,19 @@
...
-    source = DrvScpiHandlerC(port = '/dev/ttyACM0', separator = '\n', baudrate = 9600)
+    source = DrvScpiHandlerC(port = 'COM5', separator = '\n', baudrate = 9600)
     print("source")
-    # source.send_msg('SYSTem:LOCK: ON')
-    # source.send_msg('SYSTem:LOCK: OFF')
-    # source.send_and_read('MEASure:VOLTage?')
+    # # source.send_msg('SYSTem:LOCK: ON')
+    # # source.send_msg('SYSTem:LOCK: OFF')
+    print(source.send_and_read('MEASure:VOLTage?'))
     print(source.read_device_info())
```
Comment (RobertoAldea) [inline code/drv/drv_scpi/example.py:46]: Try to use logger instead of prints

**Review by RobertoAldea — CHANGES_REQUESTED** [review]

**Review by RobertoAldea — CHANGES_REQUESTED** [review]

File: code/drv/drv_scpi/example.py (hunk 2 above)
Comment (luroche) [reply code/drv/drv_scpi/example.py:19]: Done.

File: code/drv/drv_scpi/example.py (hunk 3 above)
Comment (luroche) [reply code/drv/drv_scpi/example.py:46]: Done.

File: code/drv/drv_scpi/drv_scpi.py [hunk 4]
```diff
@@ -1,35 +1,31 @@
...
+log = sys_log_logger_get_module_logger(__name__)
 
 
 #######################          PROJECT IMPORTS         #######################
 
 
+#######################          MODULE IMPORTS          #######################
+
 #######################              ENUMS               #######################
 class _DefaultSerialParamsC:
```
[the remark this answers is not in the document]
Comment (luroche) [reply code/drv/drv_scpi/drv_scpi.py:30]: @mariuscrsn

**Pull request closed without merging**